src/js/htmlutils.js

```diff
@@ -179,9 +180,10 @@ var htmlUtils = exports.htmlUtils = {
     var originHtml = '' +
       '<div class="' + classes.join(' ') + '" data-origin="' + origin + '" data-original-action="' + action + '">' +
       '<div class="origin tooltip" title="' + actionDescription + '">' + whitelistedText + origin + '</div>' +
-      '<div class="removeOrigin">&#10006</div>' +
```

Comment:
A functional test (`OptionsPageTest::test_removed_origin_display`) fails now ... Maybe some HTML reference needs to be updated, or maybe we actually did break something.

Reply:
Just updated the xpath in the test that referenced the "removeOrigin"-element;
I Wasn't able to run the test locally though, as the tests had a hard time detecting the browsers I attempted to set on my Mac...